src/madhavi_Khasbage/MethodReturnType/Calculator.java

```diff
@@ -0,0 +1,64 @@
+package madhavi_Khasbage.MethodReturnType;
+
+/*Write a class called calculator with following functionality.
+	a. add
+	b. subtraction
+	c. division
+	d. multiplication
+	e. printTotalAnswer.	
+	Method a,b,c,d should have 2 parameters and return int value.
+	printTotalAnswer method should have 4 parameters and doesnt return anything.
+	Note : do not create any instance variable.*/
+
+public class Calculator {	
+	
+	int addition(int num1, int num2){
+		int total=0;		
+		if(num1>0 && num2>0) {
```

Comment:
We can do addition of any numbers, no requirement that number have only positive only.